packages/wonder-blocks-grid/components/row.test.js

```diff
@@ -112,8 +116,10 @@ describe("Row", () => {
 
             // Prevent jest complaining when JSDOM console.error's an exception
             jest.spyOn(console, "error");
-            // eslint-disable-next-line no-console
+            /* eslint-disable no-console */
```

Comment:
Do you think is a good idea removing these lines by chaining `mockImplementation` to `jest.spyOn`?

e.g `jest.spyOn(console, "error").mockImplementationOnce(() => {});`